src/modules/agents/ui/components/agent-form.tsx

```
@@ -0,0 +1,137 @@
+  onSuccess,
+  onCancel,
+  initialValues,
+}: AgentFormProps) => {
+  const trpc = useTRPC();
+  const router = useRouter();
+  const queryClient = useQueryClient();
+
+  const createAgent = useMutation(
+    trpc.agents.create.mutationOptions({
+      onSuccess: () => {
+        onSuccess: () => {
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)